fideslib/oauth/oauth_util.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+from jose import jwe
+
+
+def extract_payload(jwe_string: str, encryption_key: str) -> str:
+    """Given a jwe, extracts the payload and returns it in string form."""
+    return jwe.decrypt(jwe_string, encryption_key)
+
+
+def is_token_expired(issued_at: datetime | None, token_duration_min: int) -> bool:
+    """Returns True if the datetime is earlier than token_duration_min ago."""
+    if not issued_at:
+        return True
+
+    return (datetime.now() - issued_at).total_seconds() / 60.0 > token_duration_min
```

Comment:
This is because the database was required for the test that was in `fidesops`. This seems like it should require the database though so let me see if I can add one.

Reply:
it still seems like a revised test could be added just to protect the functionality, no need for the db

Reply:
Yes, if I could type...I mean seems like it shouldn't be required 😄 

Working on adding one now.